lib/main.coffee

```diff
@@ -101,8 +101,24 @@ module.exports =
             return [] unless result.files[filePath]
             messages = result.files[filePath].messages
           return messages.map (message) =>
-            startPoint = [message.line - 1, message.column - 1]
-            endPoint = [message.line - 1, message.column]
+
+            # fix column in line with tabs
+            column = message.column
+            if @tabWidth > 1
```

Comment:
Heh, that makes sense as well. I guess there is no point in doing the extra work if it's going to come up with the same result 😉.